spek-junit-platform-engine/src/main/kotlin/org/jetbrains/spek/engine/Scope.kt

```diff
@@ -87,8 +90,9 @@ sealed class Scope(uniqueId: UniqueId, val pending: Pending, val source: TestSou
             getParent().get() as GroupScope
         }
 
-        override fun isTest() = true
-        override fun isContainer() = false
+        override fun getType() = TEST
+        override fun isTest() = type.isTest
+        override fun isContainer() = type.isContainer
```

Comment:
Move `isTest()` and `isContainer()` overrides to `Scope`, so you don't have to duplicate the code for each scope type.